client/app/private-database/state/private-database-state.html

```diff
@@ -6,382 +6,272 @@
+                                    </oui-action-menu-item>
+                                </oui-action-menu>
+                            </oui-tile-definition>
+                            <oui-tile-definition data-term="{{ ::'privateDatabase_dashboard_ram' | translate}}">
+                                <oui-tile-description>
+                                    <div class="oui-status"
+                                         data-ng-bind="::stateCtrl.database.ram.value + ('unit_size_' + stateCtrl.database.ram.unit | translate)"
+                                         data-ng-class="stateCtrl.database.oom.realList.length >= stateCtrl.database.oom.nbOomError ? 'oui-status_error' : (stateCtrl.database.oom.realList.length > 0 ? 'oui-status_warning' : 'oui-status_success')">
+                                   </div>
+                                   <button class="btn btn-icon btn-xs d-block text-wrap text-left" type="button" title="{{:: 'privateDatabase_modale_oom_title' | translate }}"
+                                           data-ng-class="stateCtrl.database.oom.realList.length > stateCtrl.database.oom.nbOomError ? 'text-danger' : (stateCtrl.database.oom.realList.length > 0 ? 'text-warning' : '')"
+                                           data-ng-click="stateCtrl.goToOom()">
+                                       <span data-ng-if="stateCtrl.database.oom.realList.length > 0" class="fa fa-exclamation-triangle" aria-hidden="true"></span>
+                                       <span data-ng-if="stateCtrl.database.oom.realList.length === 0" class="fa fa-check-circle" aria-hidden="true"></span>
+                                       <span data-ng-if="stateCtrl.database.oom.realList.length > 0" data-ng-bind-html="'privateDatabase_warning_oom' | translate: { t0: stateCtrl.database.oom.realList.length }"></span>
+                                       <span data-ng-if="stateCtrl.database.oom.realList.length === 0" data-ng-bind-html="'privateDatabase_no_oom' | translate"></span>
+                                   </button>
+                                </oui-tile-description>
+                                <oui-action-menu data-compact data-align="end"
+                                                 data-ng-if="!stateCtrl.isExpired">
+                                    <oui-action-menu-item data-on-click="setAction('database/ram/update/private-database-database-ram-update', stateCtrl.database)"
+                                                          data-ng-disabled="taskState.lockAction">
+                                        <span data-translate="privateDatabase_change_ram_title"></span>
+                                    </oui-action-menu-item>
+                                    <oui-action-menu-item data-on-click="stateCtrl.goToOom()"
+                                                          data-ng-disabled="taskState.lockAction">
```

Comment:
```suggestion
                                                          data-disabled="taskState.lockAction">
```
According to documentation